packages/registration-form/src/components/terms-conditions-modal/terms.tsx

```diff
@@ -0,0 +1,864 @@
+import React from 'react';
+import { gql, useQuery } from '@apollo/client';
+import { isEmpty } from 'lodash';
+import { useTermsAndConditionsContext } from '../terms-conditions/terms-conditions';
+import parse from 'html-react-parser';
+
+const GlobalTerms = ({ globalTerms }: { globalTerms: string }): JSX.Element => {
+  const query = gql`
+    query CompanyDetailsQuery {
+      CompanyDetails {
+        __typename
+        name
+      }
+    }
+  `;
+
+  const { data } = useQuery(query);
+  const companyName = data && data.CompanyDetails.name;
+  // Apply tailwind stylings to global terms
+  if (globalTerms.includes('<ol>')) {
```

Comment:
Have you explored `html-react-parser`'s `replace` function here?

https://github.com/remarkablemark/html-react-parser#replace

My concern with this approach is that we could be erasing any attributes that may have been added previously (as unlikely as they may be)

Reply:
@rjschill87 I was able to utilize the `replace` function to check for existing attributes.